audit-logs/charts/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+appVersion: v0.116.0
+name: audit-logs-operator
+version: 0.0.1
+description: OpenTelemetry Operator Helm chart for Kubernetes
+icon: https://raw.githubusercontent.com/cncf/artwork/a718fa97fffec1b9fd14147682e9e3ac0c8817cb/projects/opentelemetry/icon/color/opentelemetry-icon-color.png
+type: application
+maintainers:
+- name: timojohlo
+- name: kuckkuck
+- name: viennaa
+- name: olandr
+sources:
+- https://github.com/cloudoperators/greenhouse-extensions
+dependencies:
+- name: opentelemetry-operator
+  alias: audit-logs-operator
+  repository: https://open-telemetry.github.io/opentelemetry-helm-charts
+  version: 0.74.2
```

Comment:
This means both the `logs` and `audit-logs` Plugin will bring the whole `opentelemetry-operator`?
How do you make sure the two operator will not interfere with each other?